lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -359,31 +366,30 @@ private Map<IpvJourneyTypes, StateMachine> loadStateMachines(
         return stateMachinesMap;
     }
 
-    private void sendMitigationStartAuditEvent(
-            AuditEventUser auditEventUser, String mitigationType, String deviceInformation)
-            throws SqsException {
-
-        auditService.sendAuditEvent(
-                new AuditEvent(
-                        AuditEventTypes.IPV_MITIGATION_START,
-                        configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
-                        auditEventUser,
-                        new AuditExtensionMitigationType(mitigationType),
-                        new AuditRestrictedDeviceInformation(deviceInformation)));
-    }
-
     private void sendJourneyAuditEvent(
-            AuditEventTypes auditEventType, AuditEventUser auditEventUser, String deviceInformation)
+            AuditEventTypes auditEventType,
+            Map<String, String> auditContext,
+            AuditEventUser auditEventUser,
+            String deviceInformation)
             throws SqsException {
-
         auditService.sendAuditEvent(
                 new AuditEvent(
                         auditEventType,
                         configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
                         auditEventUser,
+                        getAuditExtensions(auditEventType, auditContext),
                         new AuditRestrictedDeviceInformation(deviceInformation)));
     }
 
+    private AuditExtensions getAuditExtensions(
+            AuditEventTypes auditEventType, Map<String, String> auditContext) {
+        return switch (auditEventType) {
+            case IPV_MITIGATION_START -> new AuditExtensionMitigationType(
+                    auditContext.get("mitigationType"));
+            default -> null;
+        };
```

Comment:
Intention here is that we have a relatively flexible system for adding metadata to audit events - we already need it for the mitigation type, but will also want it for the COI change type, etc.